### ISSUE TITLE
app broken, for give me if not a appdaemon issue. beyond my scope of knowledge. 

### ISSUE THREAD

**Issue by stovedoctor** [hassio-addons/app-appdaemon#21]
[09:40:41] INFO: Starting AppDaemon...
2020-03-08 09:40:43.689160 INFO AppDaemon: AppDaemon Version 4.0.3 starting
2020-03-08 09:40:43.689594 INFO AppDaemon: Python version is 3.8.2
2020-03-08 09:40:43.690106 INFO AppDaemon: Configuration read from: /config/appdaemon/appdaemon.yaml
2020-03-08 09:40:43.690572 INFO AppDaemon: Added log: AppDaemon
2020-03-08 09:40:43.691059 INFO AppDaemon: Added log: Error
2020-03-08 09:40:43.691522 INFO AppDaemon: Added log: Access
2020-03-08 09:40:43.691961 INFO AppDaemon: Added log: Diag
2020-03-08 09:40:43.728895 INFO AppDaemon: Loading Plugin HASS using class HassPlugin from module hassplugin
2020-03-08 09:40:43.751442 INFO HASS: HASS Plugin Initializing
2020-03-08 09:40:43.752098 INFO HASS: HASS Plugin initialization complete
2020-03-08 09:40:43.753266 INFO AppDaemon: Initializing HTTP
2020-03-08 09:40:43.754373 INFO AppDaemon: Using 'ws' for event stream
2020-03-08 09:40:43.780798 INFO AppDaemon: Starting API
2020-03-08 09:40:43.790707 INFO AppDaemon: Starting Admin Interface
2020-03-08 09:40:43.791632 INFO AppDaemon: Starting Dashboards
2020-03-08 09:40:43.816078 INFO HASS: Connected to Home Assistant 0.106.5
2020-03-08 09:40:43.877473 INFO AppDaemon: App 'schedy_heating' added
2020-03-08 09:40:43.879716 INFO AppDaemon: App 'hello_world' added
2020-03-08 09:40:43.882568 INFO AppDaemon: Found 2 total apps
2020-03-08 09:40:43.884270 INFO AppDaemon: Starting Apps with 2 workers and 2 pins
2020-03-08 09:40:43.888420 INFO AppDaemon: Running on port 
2020-03-08 09:40:43.901168 WARNING AppDaemon: ------------------------------------------------------------
2020-03-08 09:40:43.903016 WARNING AppDaemon: Unexpected error during run()
2020-03-08 09:40:43.904715 WARNING AppDaemon: ------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/appdaemon/__main__.py", line 135, in run
    loop.run_until_complete(asyncio.gather(*pending))
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/lib/python3.8/site-packages/appdaemon/utility_loop.py", line 61, in loop
    await self.AD.http.start_server()
  File "/usr/lib/python3.8/site-packages/appdaemon/http.py", line 301, in start_server
    site = web.TCPSite(self.runner, '0.0.0.0', int(self.port))
ValueError: invalid literal for int() with base 10: ''
2020-03-08 09:40:43.914105 INFO AppDaemon: Previous message repeated 1 times
2020-03-08 09:40:43.915036 INFO AppDaemon: AppDaemon Exited
/usr/lib/python3.8/site-packages/appdaemon/__main__.py:134: DeprecationWarning: Task.all_tasks() is deprecated, use asyncio.all_tasks() instead
  pending = asyncio.Task.all_tasks()
/usr/lib/python3.8/site-packages/aiohttp/connector.py:964: DeprecationWarning: The loop argument is deprecated since Python 3.8, and scheduled for removal in Python 3.10.
  hosts = await asyncio.shield(self._resolve_host(
/usr/lib/python3.8/site-packages/aiohttp/locks.py:21: DeprecationWarning: The loop argument is deprecated since Python 3.8, and scheduled for removal in Python 3.10.
  self._event = asyncio.Event(loop=loop)
[09:40:44] INFO: Starting AppDaemon...# Problem/Motivation

**Comment by frenck** [comment]
Please share your configuration, because there isn't much I can do with the provided logs.

**Comment by stovedoctor** [comment]
> Please share your configuration, because there isn't much I can do with the provided logs.

thank you for all you do!
I backed up my files and uninstalled and started over all good.
poking around in my backup files I found that somehow (I have not touched the files before crash other than a supervisor and core update.)
the URL in the appdaemon.yaml  had a added httpp://
like this.
 http://http://192.168.86.47:5050
so for future reference the F@#$%^& URL as to be correct.
again thank you for all your work and brilliant add one and help you give to DUMBSHITS like me.
